lumibot/brokers/broker.py

```diff
@@ -353,6 +353,40 @@ def get_chain(self, chains, exchange="SMART") -> dict:
         """
         return chains[exchange] if exchange in chains else chains
 
+    def get_chain_full_info(self, asset: Asset, expiry: str, chains=None, underlying_price=None, risk_free_rate=None,
+                            strike_min=None, strike_max=None) -> pd.DataFrame:
```

Comment:
Not sure if this is needed or not.  Is the error handling the same as the other similar methods?  If not then I'd suggest making the change needed to do that, otherwise I think this is N/A.